mon-pix/tests/unit/components/certification-instructions/steps_test.js

```diff
@@ -0,0 +1,60 @@
+import { module, test } from 'qunit';
+import sinon from 'sinon';
+
+import createGlimmerComponent from '../../../helpers/create-glimmer-component';
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
+
+module('Unit | Component | certification-instruction | steps', function (hooks) {
+  setupIntlRenderingTest(hooks);
+
+  module('#nextStep', function () {
+    module('when pageId is lower than pageCount', function () {
+      test('should change the pageId', async function (assert) {
+        // given
+        const component = createGlimmerComponent('certification-instructions/steps');
+
+        component.pageId = 1;
+        component.pageCount = 2;
+        component.isConfirmationCheckboxChecked = false;
+
+        // when
+        await component.nextStep();
+
+        // then
+        assert.strictEqual(component.pageId, 2);
+      });
+    });
+
+    module('when pageId equal to lower than pageCount', function () {
```

Comment:
`when pageId equal to lower than pageCount` Ca n'a pas trop de sens.
`when pageId is equal to pageCount` ?